dmutils/content_loader.py

```diff
@@ -432,6 +458,20 @@ def _populate_section(self, framework_slug, question_set, section):
         ]
         return section
 
+    def _message_key(
+        self, block, framework_status, supplier_status
+    ):
+        return '{}-{}{}'.format(
+            block,
+            framework_status,
+            '-{}'.format(supplier_status) if supplier_status else ''
+        )
+
+    def _message_error(self, framework_slug, message):
+        self._content[framework_slug]['messages'][message] = None
```

Comment:
Why is this needed? No one should be catching the `ContentNotFoundError`, so the process will termniate once the exception is thrown.